templates/custom/api.mustache

```diff
@@ -5,8 +5,9 @@ package {{packageName}}
 
 {{#operations}}
 import (
-	_context "context"
```

Comment:
Why did we remove the `_`?

Reply:
It was not really necessary, we don't have any conflict with the original `context` name.